node/sync.go

```diff
@@ -252,13 +252,6 @@ func (d *Pegnetd) SyncBlock(ctx context.Context, tx *sql.Tx, height uint32) erro
 		}
 	}
 
-	// 3) Apply FCT --> pFCT burns that happened in this block
-	//    These funds will be available for transactions and conversions executed in the next block
-	// TODO: Check the order of operations on this and what block to add burns from.
-	if err := d.ApplyFactoidBlock(ctx, tx, dblock); err != nil {
```

Comment:
This needs to be handled with activation heights so that legacy burns are still parsed.